scripts/etc/init.d/rcS

```diff
@@ -85,6 +85,25 @@ sanitize_inputfile()
     fi
 }
 
+install_files()
+{
+  file_to_read=$1
+  sanitize_inputfile file_to_read
```

Comment:
> that is/should be IMO a separate issue

agree, it will be... :wink:   (that's why I thought it wouldn't hurt if you change it now)